CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-19.11.2
+
+* CLI
+  * Updated `uninject` command to uninject namespace resources
+    (thanks @mayankshah1607!)
+* Controller
+  * Added `conntrack` to the `debug` container to help with connection tracking
```

Comment:
TIOLI: Somehow `connection tracking debugging` feels like a mouthful. What about `to allow/enable connection tracking`